linode/manager
# feat: [M3-7806] - Linode Create Refactor - Part 1

## Description 📝

This begins the refactor of the Linode Create flow. The goal is to build a new create flow from scratch using modern practices so that the Linode create flow is better and more maintainable. The goal is for this new create flow to be "backwards compatible", meaning we can flip the feature flags and customers have the exact same experience. 

- Initial PR of the Linode Create Refactor 🎉 
  - Introduces `react-hook-form` 🔘 
    - We are doing this for the type-safety, performance, and flexibility (in comparison to formik)
  - Setups up initial scaffolding and feature flag 🚩
  - Begins incorporating some existing components (region select and plan selection)

## Preview 📷

https://github.com/linode/manager/assets/115251059/40b40d5e-9b59-452c-b276-c2a9b3be1c82

## How to test 🧪

- Check out this PR
- Use our local dev tools 🔧 and enable the new create flow feature flag
- Test the new create flow
- Look at my code and be critical
- Verify I didn't break anything with the existing Linode create flow components

## As an Author I have considered 🤔
